Cake.XComponent.Test/Cake.XComponent.Test.csproj

```diff
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Cake.XComponent.Test</RootNamespace>
     <AssemblyName>Cake.XComponent.Test</AssemblyName>
-    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.1</TargetFrameworkVersion>
```

Comment:
Why not netstandard2.0 ?

Reply:
Changing to netstandard2.0 is a bit painful.
And this way we check that it works from 4.6.1